spread/runner.go

```diff
@@ -82,6 +82,8 @@ func Start(project *Project, options *Options) (*Runner, error) {
 			r.providers[bname] = Linode(project, backend, options)
 		case "lxd":
 			r.providers[bname] = LXD(project, backend, options)
+		case "lxd-vm":
+			r.providers[bname] = LXDVM(project, backend, options)
```

Comment:
Can you document the new backend in `README.md`?

Reply:
done